Phoenix/PHSpace.m

```diff
@@ -36,6 +37,9 @@ @implementation PHSpace
 static NSString *const CGSSpaceIDKey = @"ManagedSpaceID";
 static NSString *const CGSSpacesKey = @"Spaces";
 
+// An arbitrary ID we can use CGSMoveWindowsToManagedSpace with.
+const NSUInteger MoveWindowsCompatId = 0x79616265;
```

Comment:
Couldn’t we typedef this as `CGSMoveWindowCompatID` or something?

Reply:
>Can you make this static?

I didn't know static worked that way in Objective C, cool - done!

>Do we need this to be a hex value?

My guess is that it's an arbitrary ID, but I'm not sure if there's anything special about it - it's just the same one from the yabai and Amethyst fixes, and I figured it would be most consistent to use the same one.

>Couldn’t we typedef this as `CGSMoveWindowCompatID` or something?

I'm not sure if that's the official name, but happy to!